requirements/static/py3.7/windows.txt

```diff
@@ -72,7 +72,7 @@ pyaml==19.4.1             # via moto
 pyasn1-modules==0.2.4     # via google-auth
 pyasn1==0.4.5
 pycparser==2.19
-pycryptodome==3.8.1       # via python-jose
+pycryptodome==3.9.8
 pycryptodomex==3.9.7
```

Comment:
Isn't it `pycryptodomex` that get's bundled with the Windows releases, not `pycryptodome`?  Should it be updated as well?